Cargo.toml

```diff
@@ -33,16 +33,18 @@ name = "kv"
 harness = false
 
 [features]
-default = ["vendored-duckdb", "vendored-sqlite"]
+default = ["vendored-duckdb", "vendored-sqlite", "vendored-rocksdb"]
 binary = [
     "dep:clap",
     "dep:tracing-subscriber", # no way to make this contingent on tracing also being enabled
 ]
 duckdb = ["dep:duckdb"]
 sqlite = ["dep:rusqlite"]
+rocksdb = ["dep:rocksdb", "dep:rand"]
```

Comment:
I thought replacing every #[cfg(feature = "rocksdb")] with #[cfg(feature = "vendored-rocksdb")] doesn't seem nice? Or if you don't mind, we can go that route.

Reply:
In `build.rs`, you can add the following

```rust
if cfg!(feature = "vendored-rocksdb") {
    println!("cargo::rustc-cfg=feature=\"rocksdb\"");
}
```

This will mean that you can use `feature = "rocksdb"` within the crate without actually having a `rocksdb` feature.